File: sel py/a/navigation.py
from selenium import webdriver
from selenium.webdriver.common.keys import Keys
import time

driver = webdriver.Chrome(executable_path="C:/Users/maruf/Desktop/prac/webdriver/chromedriver.exe")
# driver = webdriver.Firefox(executable_path="C:/Users/maruf/Desktop/prac/webdriver/geckodriver.exe")

s = "http://login.facebook.com/"
driver.get(s)
time.sleep(5)
driver.maximize_window()
print(driver.title)
# print(driver.current_url)
# print(driver.page_source)
#driver.find_element_by_xpath('//*[@id="u_0_b"]').click()
driver.get("http://google.com/")
time.sleep(5)
driver.get("http://yahoo.com")
driver.back()
time.sleep(5)
driver.forward()
time.sleep(5)
driver.forward()
time.sleep(5)
driver.quit()

File: sel java/a/src/q/navigation.java
package q;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.By;
import org.openqa.selenium.firefox.FirefoxDriver;

import java.util.concurrent.TimeUnit;

public class navigation {
    public static void main(String[] args) {
        //System.setProperty("webdriver.chrome.driver","C:\\Users\\maruf\\Desktop\\prac\\webdriver\\chromedriver.exe");
        System.setProperty("webdriver.gecko.driver",
                "C:\\Users\\maruf\\Desktop\\prac\\webdriver\\geckodriver.exe");
        WebDriver d= new FirefoxDriver();
        d.manage().window().maximize();
        String s="http://facebook.com/";
        //d.get(s);
        d.navigate().to(s);
        System.out.println(d.getTitle());
        d.findElement(By.xpath("//*[@id=\"u_0_b\"]")).click();
        d.get("http://google.com/");
        System.out.println(d.getTitle());
        d.navigate().back();
        d.close();
    }
}
